### PR TITLE
chore: add release-please

### LINKED ISSUE #215: Add release-please

### PR DESCRIPTION
**Pull request by spaenleh**
This PR adds release-please.

closes #215

### CODE REVIEW

File: .github/workflows/release-please.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - uses: actions/checkout@v3
+
+      # in case a release was created, this will create or update the major and minor tags
+      # so you can refer to `v1` or `v1.1` and get the latest version corresponding to that prefix.
+      - name: Tag major and minor versions
+        uses: jacobsvante/tag-major-minor-action@v0.1
+        if: ${{ steps.release.outputs.release_created }}
+        with:
+          major: ${{ steps.release.outputs.major }}
+          minor: ${{ steps.release.outputs.minor }}
```
Comment (dialexo) [inline .github/workflows/release-please.yml:32]: By the way would it make sense to factor this out into a graasp composite action? I'm not sure how similar it is to the other release-please workflows. Let me know if you think it's worth it!

**Review by dialexo — APPROVED** [review]
LGTM ✅

File: .github/workflows/release-please.yml (hunk 1 above)
Comment (spaenleh) [reply .github/workflows/release-please.yml:32]: @dialexo In fact, this workflow is used as-is in a lot of projects but i dont know if refactoring it into a composite makes more sense ... here i can just create it from the repo workflows suggested by GitHub. Abstracting it into a composite would make it more difficult to adapt, I don't know if the trade-of is worth it. We can discuss during the stand-up with @pyphilia and @juancarlosfarah

Comment (dialexo) [reply .github/workflows/release-please.yml:32]: Sounds good, I think you know the best 😊

**Pull request merged**